src/plusplus.coffee

```diff
@@ -51,10 +51,15 @@ module.exports = (robot) ->
 
     # do some sanitizing
     reason = reason?.trim().toLowerCase()
-    name = (name.replace /(^\s*@)|([,:\s]*$)/g, "").trim().toLowerCase() if name
+
+    if name
+      if name.charAt(0) == ":"
+        name = (name.replace /(^\s*@)|([,\s]*$)/g, '').trim().toLowerCase()
+      else
+        name = (name.replace /(^\s*@)|([,:\s]*$)/g, '').trim().toLowerCase()
 
     # check whether a name was specified. use MRU if not
-    unless name? && name != ''
+    unless name? && name != ""
```

Comment:
Only for the aesthetic. Noticed that double quotes were used in most other places -- although in doing a more thorough check I see it's more of a 50/50 mix. I can swap those back.